codespell_lib/data/dictionary.txt

```diff
@@ -13809,6 +13831,7 @@ pounts->points
 poverful->powerful
 poweful->powerful
 powerfull->powerful
+powerppc->powerpc
```

Comment:
Specific but relevant, although I wonder if the correction should be capitalised.

Reply:
You tell me

Reply:
What was the context you found it in? Officially it is, but possibly not in a code context: https://en.wikipedia.org/wiki/PowerPC

I'd be tempted to say yes, and people can manually lower case it if required by code.

Reply:
This is the way is written as flag for configure (autogen) and so on.